subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -158,6 +158,32 @@ config NRF_CLOUD_GATEWAY
 	  Enables functionality in this device to be compatible with
 	  nRF Cloud LTE gateway support.
 
+config NRF_CLOUD_DOWNLOADS
+	bool
+	default y
+	depends on NRF_CLOUD_FOTA || NRF_CLOUD_PGPS || NRF_CLOUD_FOTA_POLL
+	help
+	  Enable nRF Cloud file downloads for FOTA and P-GPS.
+
+config NRF_CLOUD_COAP_DOWNLOADS
+	bool "Download files using CoAP instead of HTTP"
+	depends on NRF_CLOUD_COAP
+	depends on NRF_CLOUD_DOWNLOADS
+	select DOWNLOADER_TRANSPORT_COAP
+	default y
+	help
+	  Use nRF Cloud CoAP's proxy download resource to download files for FOTA and P-GPS.
+	  When enabled, the Kconfig option COAP_EXTENDED_OPTIONS_LEN_VALUE must be increased
+	  from the default value. Safe values are 80 for P-GPS and 192 for FOTA.
+
+config NRF_CLOUD_HTTPS_DOWNLOADS
+	bool "Download files using HTTPS instead of HTTP"
+	depends on !NRF_CLOUD_COAP_DOWNLOADS
+	depends on NRF_CLOUD_DOWNLOADS
+	default y
+	help
+	  Use HTTPS to download files for FOTA and P-GPS.
```

Comment:
why is this not a choice with the following possibilities:
- COAP
- HTTP
- HTTPS

Using a choice will also make the selection options mutual exclusive/

Reply:
http was never an option as it turns out